### PR TITLE
Simplify ProductPagesApi interface

### PR DESCRIPTION
**Pull request by hluk**
JIRA: RHELWF-10977

### CODE REVIEW

File: src/retasc/product_pages_api.py [hunk 1]
```diff
@@ -14,66 +15,32 @@ class ProductPagesApi:
...
-        phase__lt=Unsupported means "any supported phase" (these are less than Unsupported).
-        """
-
+    def active_releases(self, product_shortname: str) -> list[str]:
+        """Gets list of active release names."""
         opt = {
             "product__shortname": product_shortname,
             "phase__lt": "Unsupported",
-            "fields": fields,
+            "fields": "shortname",
```
Comment (zhanglu1122) [inline src/retasc/product_pages_api.py:27]: `ga_date,phase_display` think we will need those two fields in future, but can be added when it needs

**Review by zhanglu1122 — COMMENTED** [review]
comment inline

File: src/retasc/product_pages_api.py (hunk 1 above)
Comment (hluk) [reply src/retasc/product_pages_api.py:27]: Why would we need those? I think "GA" task with the date is always part of the schedule. Not sure how would we use `phase_display`.

**Comment by qgong** [comment]
LGTM!

**Review by qgong — APPROVED** [review]

File: src/retasc/product_pages_api.py (hunk 1 above)
Comment (zhanglu1122) [reply src/retasc/product_pages_api.py:27]: When create the spreadsheet, but they might need or not need in future. Remove them are fine now.
My concern is if you need to add some fields, you need to update the code, which makes it not very flexible.

Comment (hluk) [reply src/retasc/product_pages_api.py:27]: Let's keep it simpler for now. I was thinking about passing the dict with schedules to the templates so you would be able to refer to them (something like `{{ schedules["GA"].start_date }}`).

**Pull request merged**